drupalux/claro
# Apply text field style update

## Issue

This PR fixes #7. [Link to the issue](https://github.com/drupalux/seven/issues/7)

## Screenshot / UI changes

Increased font size globally to 16px/1.5 (this is the size that text fields are prepared to).
Every brand color is included as variable. 

[Screenshots (zip)](https://github.com/drupalux/seven/files/2571430/Seven.text-input.zip)

## Testing instructions

* Add `https://github.com/zolhorvath/seven-refresh/` as `drupal-profile` (`sevenrefresh`)
* Remove `core/themes/seven`
* Make sure that this theme is installed to `themes/custom/seven`
* Symlink `core/misc` to `themes/custom/misc`
* Run `yarn && yarn build:css` in the theme's root
* Install a Drupal with profile `sevenrefresh`. This will enable additional field types such as `telephone` or `date_range` and enable `styleguide` drupal module as well

After that Drupal is installed, Styleguide will be the front page and you'll have the test form at `/contact/testform`.
